### PR TITLE
adding C++ language

### PR DESCRIPTION
**Pull request by Panquesito7**
Things added/changed:

- Add the C++ language to the list.

### CODE REVIEW

**Comment by Panquesito7** [comment]
@amino19, ping. Nobody is watching this repository. 👀

**Comment by amino19** [comment]
> @amino19, ping. Nobody is watching this repository. eyes

@Panquesito7: Actually we do, but working at back-end, also we consider `installation.md` in here! 
Confused?

File: lang/cpp.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+```c++
+#include <iostream>
+
+int main(){
+    std::cout << "Hello world!\n";
+    return 0;
+}
+```
+
+# C++ is the most powerful language!
```
Comment (amino19) [inline lang/cpp.md:17]:
```suggestion
# C++ is the most powerful language!
```
One has to write the whole installation process reg. Compiler, IDE, & extensions of VSC (Visual Studio Code), I am making one for reference, so give me some time.. : )

**Review by amino19 — CHANGES_REQUESTED** [review]

**Comment by Panquesito7** [comment]
Should I start working on this PR again, or is someone working on it? Thanks. 🙂

**Comment by amino19** [comment]
> Should I start working on this PR again, or is someone working on it? Thanks. slightly_smiling_face

Yes, obv! :slightly_smiling_face: 
Pl start contributing as it follows [here](https://github.com/devstrons/hello-world/blob/87c0b886b781edd21d053ee389a615895c48a013/lang/java.md).

**Comment by amino19** [comment]
@Panquesito7: I hope you're not forgotten about this Open Source Journey, emphasis on you isn't a good step because I believe you are already passionate about Open Source contributions, pl make it done, or assign anyone who completes this for you! :slightly_smiling_face:

**Comment by Panquesito7** [comment]
Working on this (sorry for the very very long delay).

**Review by aminoxix — APPROVED** [review]
LGTM! 🥳

**Pull request merged**